typescript-conditionals/main.ts

```diff
@@ -0,0 +1,117 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+interface Person {
+  name: string;
+  age: number;
+}
+
+function isUnderFive(number: number): boolean {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('isUnderFive:', isUnderFive(10));
+
+function isEven(number: number): boolean {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+console.log('isEven:', isEven(8));
+
+function startsWithJ(string: string): boolean {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('startsWithJ:', startsWithJ('Shea'));
+
+function isOldEnoughToDrink(person: Person): any {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+const shea = {
+  name: 'Shea Lindsay Pone',
+  age: 27,
+};
+console.log('isOldEnoughToDrink:', isOldEnoughToDrink(shea));
+
+function isOldEnoughToDrive(person: Person): any {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('isOldEnoughToDrive:', isOldEnoughToDrive(shea));
+
+function isOldEnoughToDrinkAndDrive(person: Person): any {
+  if (person.age >= 0) {
```

Comment:
they want you to use AND operator for the two conditions (drinking and driving).